tests/utils/test_query_limit.py

```diff
@@ -0,0 +1,41 @@
+import unittest
```

Comment:
This file test only contains for single query, multi query test is in test_basesql_queryrunner.py